topdown/crypto.go

```diff
@@ -126,6 +131,45 @@ func builtinCryptoX509ParseCertificateRequest(a ast.Value) (ast.Value, error) {
 	return ast.InterfaceToValue(x)
 }
 
+func builtinCryptoX509ParseRSAPrivateKey(_ BuiltinContext, args []*ast.Term, iter func(*ast.Term) error) error {
+
+	a := args[0].Value
+	input, err := builtins.StringOperand(a, 1)
+	if err != nil {
+		return err
+	}
+
+	invalid := ast.BooleanTerm(false)
```

Comment:
Hmm I think we might better return an error here? That way, the builtin call becomes undefined, and the error will be visible if running with strict-builtin-errors, or when #3742 is implemented.

Reply:
do you mean instead of using "ast.BooleanTerm", to use a "ast.StringTerm" with a more cleared error message in the following err condition block?

Reply:
Not exactly, I mean the built-in should fail. The common way to do that is to return an error. If it's not a halt error, evaluation will continue, the calls value is undefined.

Reply:
I see, so instead of returning the ast boolean term, return the err directly as "return err" in the following blocks. Will update this shortly.